Kubernetes/HOWTO-on-prem.md

```diff
@@ -59,41 +78,41 @@ In the `manifest` folder, run the Python script, passing your master IP and the
 This will generate a set of YAML files. You should [re]move the Python script so that K8s doesn't mistake it for a manifest and cause problems.
 
 ### Configure & Run Kubernetes ###
-Run the script `configure-kubectl.sh`, passing your external IP again, to create a configuration in `~/.kube/config` containing all of the certificate data and other parameters:
+Run the script `configure-kubectl.sh` passing your external IP again. This will create a configuration in `~/.kube/config` containing all of the certificate data and other parameters:
 
     ~/kube $ ./configure-kubectl.sh 10.123.45.67
 
-Run the script `start-kubelet.sh`. You may need to run it with `sudo`. Then, in another shell, copy the Kubernetes configuration file to the local kubelet mount path:
+Run the script `start-kubelet.sh`. You may need to run it with `sudo`. Then, in another shell, watch your working directory (`~/kube`) until the folder `kubelet/` appears. Immediately copy the Kubernetes configuration file to it:
 
-    $ sudo cp ~/.kube/config kubelet/
+    ~/kube $ sudo cp ~/.kube/config kubelet/
 
 **TODO**: Why is this necessary? Shouldn't this be done automatically by one of the containers?
 
 In another terminal session, run the Kubeproxy script, passing your cluster CIDR (you may also need to run this one with `sudo`):
 
     ~/kube $ ./kubeproxy.sh 192.168

```

Comment:
I get an error from kube-proxy complaining about conntrack not being available.
E1010 15:55:19.407842  103781 proxier.go:942] Failed to delete kube-system/kube-dns:dns endpoint connections, error: error deleting conntrack entries for UDP peer {11.0.0.10, 10.10.0.9}, error: error looking for path of conntrack: exec: "conntrack": executable file not found in $PATH

Reply:
I'm not sure how conntrack is being used, but I also saw this error. You can resolve it by apt-get'ing conntrack, so we should probably add this to the guide along with the step where we install docker etc.